### PR TITLE
Update Compliance and SDR Germany roles

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn , @jutta-frieden , @ahmedbadr3 : please review.

### CODE REVIEW

File: app/pages/about/jobs/positions/compliance-manager.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default class ComplianceManager extends React.Component {
   render() {
     return (
       <JobsPage>
-        <h3 className='u-text-heading-light u-text-m u-color-heading'>Compliance Manager </h3>
+        <h3 className='u-text-heading-light u-text-m u-color-heading'>MLRO and Compliance Manager </h3>
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/compliance-manager.js]: Stray space after "Manager"

File: app/pages/about/jobs/positions/sales-development-representative-germany.js [hunk 2]
```diff
@@ -78,10 +81,6 @@ export default class SalesDevelopmentRepresentativeGermany extends React.Compone
             Stocked kitchen and unlimited (great) coffee!
           </li>
         </ul>
-        <p className='para'>
-          However our team comes from a variety of backgrounds and
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: Why has this whole line been removed?

**Comment by jacobpgn** [comment]
There are 3 distinct changes in this PR - keep 'em separate!

**Comment by ahmedbadr3** [comment]
The compliance manager one looks good to me – thanks!

File: app/pages/about/jobs/positions/sales-development-representative-germany.js (hunk 2 above)
Comment (gabrielamatias) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: It repeated what was already said previously in the ad before the perks section, so we've decided to remove it.

Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-germany.js]: Ah I see it now, 👍 !

**Pull request merged**